benchmarks/check_xla_device.py

```diff
@@ -1,12 +1,24 @@
 import sys
-assert len(sys.argv) == 2
+
+assert len(sys.argv) in (2, 3)
```

Comment:
could we make a function `use_torch_xla2`?

Reply:
Done